ui/address/AddressDetails.tsx

```diff
@@ -91,9 +99,52 @@ const AddressDetails = ({ addressQuery, scrollRef }: Props) => {
         rowGap={{ base: 1, lg: 3 }}
         templateColumns={{ base: 'minmax(0, 1fr)', lg: 'auto minmax(0, 1fr)' }} overflow="hidden"
       >
+        { data.filecoin?.id && (
+          <>
+            <DetailsInfoItem.Label
+              hint="Short identifier of an address that may change with chain state updates"
+            >
+              ID
+            </DetailsInfoItem.Label>
+            <DetailsInfoItem.Value>
+              <Text>{ data.filecoin.id }</Text>
+              <CopyToClipboard text={ data.filecoin.id }/>
+            </DetailsInfoItem.Value>
+          </>
+        ) }
+
+        { data.filecoin?.actor_type && (
+          <>
+            <DetailsInfoItem.Label
+              hint="Identifies the purpose and behavior of the address on the Filecoin network"
+            >
+              Actor
+            </DetailsInfoItem.Label>
+            <DetailsInfoItem.Value>
+              <FilecoinActorTag actorType={ data.filecoin.actor_type }/>
+            </DetailsInfoItem.Value>
+          </>
+        ) }
+
+        { (data.filecoin?.actor_type === 'evm' || data.filecoin?.actor_type === 'ethaccount') && data?.filecoin?.robust && (
+          <>
+            <DetailsInfoItem.Label
+              hint="0x-style address to which the Filecoin address is assigned by the Ethereum Address Manager"
+            >
+              Ethereum Address
+            </DetailsInfoItem.Label>
+            <DetailsInfoItem.Value flexWrap="nowrap">
+              <Box overflow="hidden">
+                <HashStringShortenDynamic hash={ data.hash }/>
+              </Box>
+              <CopyToClipboard text={ data.hash }/>
```

Comment:
If it is an address hash, it is better to use the common `AddressEntity` component.